src/Nordea/Components/Label.elm

```diff
@@ -113,6 +89,31 @@ init labelText labelType =
 
 view : List (Attribute msg) -> List (Html msg) -> Label -> Html msg
 view attrs children (Label config) =
+    let
+        commonConfig =
+            let
+                fromLabelRequirednessHint requirednessHint =
+                    case requirednessHint of
+                        Just (Mandatory a) ->
+                            Just (Common.Mandatory a)
+
+                        Just (Optional a) ->
+                            Just (Common.Optional a)
+
+                        Just (Custom a) ->
+                            Just (Common.Custom a)
+
+                        Nothing ->
+                            Nothing
+            in
+            { labelText = config.labelText
+            , requirednessHint = config.requirednessHint |> fromLabelRequirednessHint
```

Comment:
just wondering about the wording here, requirednessHint. Would requiredHint be sufficient?

Reply:
I think this is a hint regarding the requiredness of the field. I guess requiredHint would work fine as well, but maybe that was not chosen so it should not be mistaken for being a hint that is required (?) 🤔   We actually copied that type (and wording) from `Label` 😅

Reply:
😅